sqlstore/src/main/resources/db/migration/V8900__hibernatize_everything.sql

```diff
@@ -36,8 +36,7 @@ ALTER TABLE SequencerReference ADD CONSTRAINT upgraded_SR_UK UNIQUE (upgradedSeq
 DROP TABLE Request;
 DROP TABLE Request_Note;
 
-ALTER TABLE Status ADD FOREIGN KEY (instrumentName) REFERENCES SequencerReference (name);
-ALTER TABLE Status ADD FOREIGN KEY (runName) REFERENCES Run (alias);
+ALTER TABLE Status ADD CONSTRAINT Status_SequencerReference_FK FOREIGN KEY (instrumentName) REFERENCES SequencerReference (name);
```

Comment:
Using name instead of ID as foreign key is odd enough, but the foreign key isn't even required. The instrument is already linked through the run, which is also linked by name instead of ID! :worried: 

(this is not really a change request since there's a ticket to improve it already; I just couldn't not say anything)